src/modules/dashboard/components/ActionsPage/VoteResultsWidget/VoteResultsWidget.tsx

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+
+import Icon from '~core/Icon';
+import Heading from '~core/Heading';
+import HookedUserAvatar from '~users/HookedUserAvatar';
+import { Colony } from '~data/index';
+import { Address } from '~types/index';
+import { getMainClasses } from '~utils/css';
+
+import useAvatarDisplayCounter from '../../../hooks/useAvatarDisplayCounter';
+
+import styles from './VoteResultsWidget.css';
+
+interface Appearance {
+  theme?: 'approve' | 'disapprove';
+}
+
+interface Props {
+  colony: Colony;
+  appearance?: Appearance;
+  value?: number;
+  title: string;
+  voters?: Address[];
+  maxAvatars?: number;
+}
+
+const displayName = 'VoteResultsWidget';
+
+const VoteResultsWidget = ({
+  colony: { colonyAddress },
+  appearance = { theme: 'approve' },
+  value = 0,
```

Comment:
You also need a `max` value prop otherwise you can't properly display the bar.

![Screenshot from 2021-03-12 12-22-25](https://user-images.githubusercontent.com/1193222/110927374-38d2b780-832e-11eb-82b3-b55f151172cd.png)

And it will over-run the container